src/internal/observable/dom/WebSocketSubject.ts

```diff
@@ -8,6 +8,58 @@ import { Observer, NextObserver } from '../../types';
 import { tryCatch } from '../../util/tryCatch';
 import { errorObject } from '../../util/errorObject';
 
+/**
+ * WebSocketSubjectConfig is a plain Object that allows us to make our
+ * webSocket configurable.
+ *
+ * <span class="informal">Provides flexibility to {@link webSocket}</span>
+ *
+ * It defines a set of properties to provide custom behavior in specific
+ * moments of the socket's lifecycle. When the connection opens we can
+ * use `openObserver`, when the connection is closed `closeObserver`, if we
+ * are interested in listening for data comming from server: `deserializer`,
+ * which allows us to customize the deserialization strategy of data before passing it
+ * to the socket client.
+ *
+ * ## Example
+ * By default `deserializer` is going to apply `JSON.parse` to each message comming
```

Comment:
could you put the information about the `JSON.parse` in the description block above to and therefore slightly change the description here